environment.yml

```diff
@@ -1,60 +1,38 @@
 name: geo_deep_env
 channels:
-  - anaconda
-  - conda-forge
   - pytorch
-  - defaults
+  - conda-forge
 dependencies:
-  - fiona>=1.8.0
-  - gdal>=3.0.0
   - boto3>=1.19.2
-  - cudatoolkit=10.2.89
+  - cudatoolkit>=11.3.0
+  - pytorch=1.10.1=py3.9_cuda11.3_cudnn8.2.0_0  # Linux
+  # - pytorch=1.10.1=py3.9_cuda11.3_cudnn8_0  # Windows
```

Comment:
Couldn't find a way to make it work without hardcoding this. Currently, things are also harcoded to Linux (see l21 of old version) so this PR is at least not making things worst...

Reply:
I think it's OK.  The version of PyTorch/cuda/cudnn won't change very often.

Your [package match specification](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications) is the "three-part" type (two equal signs in the string).  Documentation says:
The third part is always the exact build string. When there are 3 parts, **the second part must be the exact version**. (bold mine)

Tests this morning:
- 1 part `pytorch` ; fetches `pytorch-1.10.2|py3.9_cuda11.3_cudnn8.2.0_0`
- 2 parts; `pytorch=1.10.*1*`; fetches `pytorch-1.10.1|py3.9_cuda11.3_cudnn8.2.0_0 1.21 GB`
- 2 parts; `pytorch=1.10.*2*`; fetches `pytorch-1.10.2             |py3.9_cuda11.3_cudnn8.2.0_0 1.21 GB`

What strikes me here is how the conda package looks like it is the exact same irrespective of PyTorch version.  Since we don't want the PyTorch version to move under our feet, the 1 part package match spec is out.  I think we could use 2 parts at the current latest stable version of PyTorch, 1.10.2.